File: CS/WindowsFormsApplication32/Form1.cs
using System;
using System.Windows.Forms;
using DevExpress.XtraEditors;
using DevExpress.XtraTabbedMdi;

namespace WindowsFormsApplication32
{
	public partial class Form1 : Form
	{
		public Form1()
		{
			InitializeComponent();
            this.myTabbedMdiManager1.HeaderShowActiveFilesButton = true;
		}

		private void Form1_Load(object sender, EventArgs e)
		{
			XtraForm xtraForm = new XtraForm();
			xtraForm.MdiParent = this;
			xtraForm.Text = "Form 1";
			xtraForm.Show();

			XtraForm xtraForm1 = new XtraForm();
			xtraForm1.MdiParent = this;
			xtraForm1.Text = "Form 2";
			xtraForm1.Show();
		}

		private void myTabbedMdiManager1_PageAdded(object sender, MdiTabPageEventArgs e)
		{
			XtraTabbedMdiManager tabbedMdi = (XtraTabbedMdiManager)sender;
			e.Page.ImageIndex = tabbedMdi.Pages.Count - 1;
		}
	}
}

File: VB/WindowsFormsApplication32/Form1.vb
Imports System
Imports System.Windows.Forms
Imports DevExpress.XtraEditors
Imports DevExpress.XtraTabbedMdi

Namespace WindowsFormsApplication32

    Public Partial Class Form1
        Inherits Form

        Public Sub New()
            InitializeComponent()
            myTabbedMdiManager1.HeaderShowActiveFilesButton = True
        End Sub

        Private Sub Form1_Load(ByVal sender As Object, ByVal e As EventArgs)
            Dim xtraForm As XtraForm = New XtraForm()
            xtraForm.MdiParent = Me
            xtraForm.Text = "Form 1"
            xtraForm.Show()
            Dim xtraForm1 As XtraForm = New XtraForm()
            xtraForm1.MdiParent = Me
            xtraForm1.Text = "Form 2"
            xtraForm1.Show()
        End Sub

        Private Sub myTabbedMdiManager1_PageAdded(ByVal sender As Object, ByVal e As MdiTabPageEventArgs)
            Dim tabbedMdi As XtraTabbedMdiManager = CType(sender, XtraTabbedMdiManager)
            e.Page.ImageIndex = tabbedMdi.Pages.Count - 1
        End Sub
    End Class
End Namespace
